config.json

```diff
@@ -197,7 +197,6 @@
           ]
         },
         {        
-          "slug": "clock",
```

Comment:
I think these changes are unrelated and likely not intentional?

Reply:
They're an artefact from saving a copy of config.json that had two clock slugs and then reinstating it and then discovering that I had saved a buggy copy. Thus the fix.

Reply:
Aha! Okay cool.